### ISSUE TITLE
Close log button lacks a label

### ISSUE THREAD

**Issue by henrycatalinismith** [henrycatalinismith/zetkin#7]
## Steps to reproduce

1. Open an assignment
2. Click the log button
3. Start your screen reader
4. Focus the close button

## Expected outcome

The button should have a label describing its purpose as per [WCAG success criterion 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html).

## Actual outcome

There's no label so it's unclear what clicking it will do.

![VoiceOver with the button focused and the announcement just says button](https://user-images.githubusercontent.com/566159/191589508-505f4b5a-a7cc-470f-9299-1aec2634613c.png)

## Recommendation

Add an `aria-label` along the lines of `Close`.
